dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/FATSuite.java

```diff
@@ -33,7 +33,9 @@
                 CXFJMXSupportTest.class,
                 WebServiceMonitorTest.class,
                 HttpConduitPropertiesTest.class,
+                //HttpConduitPropertiesTest23.class,
```

Comment:
Because these tests use jaxwstest-2.x feature files to override what API's are exported to the APP class path, I've had to break the tests into different versions. I had a problem building them after I undid some tests changes, and for the sake of time I've just stashed them locally to revisit.